input_data.go

```diff
@@ -6,7 +6,9 @@ type Modifier int
 
 var (
 	Mouse mouse
+)
 
+const (
 	MOVE    = Action(0)
```

Comment:
Similarly as described above, use explicit types rather than inferred types when defining constants for keys.

```diff
- var (
- 	Dash         = Key(189)
- 	Apostrophe   = Key(222)
- 	Semicolon    = Key(186)
- 	Equals       = Key(187)
+ var (
+ 	Dash         Key = 189
+ 	Apostrophe   Key = 222
+ 	Semicolon    Key = 186
+ 	Equals       Key = 187
```